src/commcare_cloud/environment/secrets/backends/ansible_vault/tests/expected-generated-variables.yml

```diff
@@ -0,0 +1,64 @@
+AMAZON_S3_ACCESS_KEY: "{{ localsettings_private.AMAZON_S3_ACCESS_KEY | default(None) }}"
```

Comment:
Do you think in the future we'd get rid of this mapping file and just have the secrets named correctly to start with?

Reply:
Yeah I was thinking that, although there's some value in setting the default to None. I didn't really decide on an approach here for that, though I do think the split between what's in `secrets` and what's in `localsettings_private` is sort of arbitrary and we'd do well to just get rid of that namespace distinction eventually